Sources/Request Strategies/MLS/Actions/CountSelfMLSKeyPackagesActionHandler.swift

```diff
@@ -0,0 +1,68 @@
+//
+// Wire
+// Copyright (C) 2022 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import Foundation
+
+class CountSelfMLSKeyPackagesActionHandler: ActionHandler<CountSelfMLSKeyPackagesAction> {
+
+    // MARK: - Methods
+
+    override func request(
+        for action: ActionHandler<CountSelfMLSKeyPackagesAction>.Action,
+        apiVersion: APIVersion) -> ZMTransportRequest? {
```

Comment:
In the method below you are formatting it like follows:

```suggestion
    override func request(
        for action: ActionHandler<CountSelfMLSKeyPackagesAction>.Action,
        apiVersion: APIVersion
    ) -> ZMTransportRequest? {
```

I think we should be consistent and change it here.

Reply:
Definitely, its more clean and easy to read.